### ISSUE TITLE
Don't center map to position when far from Paris

### ISSUE THREAD

**Issue by eliemichel** [Phyks/VelibFxos#84]
I also suggest to disable the bike and stand view then, or at least the display of the path to reach the station as (1) it is useless and (2) it takes time to compute (for instance if you are in New York…).
- [x] Disable view centering on the geolocation when at, say, more than `Config.maxDistanceFromParis` from Paris.
- [ ] Disable direction computing when at more than `Config.maxDistanceFromStation`
- [ ] Display a status message to explain the situation, like "Vous êtes actuellement loin de Paris. Géoposition désactivée."

**Comment by Phyks** [comment]
:+1:

**Comment by Phyks** [comment]
Implemented the first point, using `Config.maxDistanceFromCity` to be city agnostic.

Not sure to understand the point of the second point though.

Did not yet take care of the status message.

**Comment by eliemichel** [comment]
Then you could have checked the corresponding box. ;)
The second point is that if I am at, say 50km from Paris, even if the view is centered on me, I don't want it to show me haw to _walk_ to any station.

**Comment by Phyks** [comment]
Oh, did not think I could actually check without editing your comment!

Indeed for the second point, but I do not want it to show me how to walk to any station. But once I selected a station, it seems ok to guide me to it, whatever the distance. If I am out of Paris, using the bikes or stands view is a nonsense. And the search view would show the roadmap only if I select a station and want some routes.

**Comment by eliemichel** [comment]
Yeah but people might want to use those views anyway, just to know which station are the closest, maybe for fun or whatever. We cannot take the risk to have the application lag.
